openapi/api.yaml

```diff
@@ -876,6 +883,7 @@ components:
               type: string
           required:
           - width
+          - auto_download
```

Comment:
Oh cool, did it catch something?

Reply:
yep it did! wasnt a bug, but was nice to validate it worked on the first try